mmv1/third_party/terraform/tests/resource_data_loss_prevention_deidentify_template_test.go

```diff
@@ -1109,3 +1109,240 @@ resource "google_data_loss_prevention_deidentify_template" "basic" {
 }
 `, context)
 }
+
+func TestAccDataLossPreventionDeidentifyTemplate_dlpDeidentifyTemplateTemplateIdUpdate(t *testing.T) {
```

Comment:
This is 100% my fault for requesting the update tests, but I think we don't need them here, because the field is `immutable` (I missed that on the first look).

Very sorry for the thrash, but I would remove both sets of handwritten `*Update` tests (in the future, if this were an updatable field, it is typically best to include a new field in the existing update test, when one exists).